containers/init/ready/ready.go

```diff
@@ -52,6 +53,14 @@ const DefaultTimeout = 25 * time.Minute
 // unset, DefaultOutputFile will be used as the default.
 const OutputFileEnv = "READY_OUTPUT_FILE"
 
+// OutputMetadataEnv is the optional name of the file where the executable should
+// write a comma-separated list of IP addresses.
```

Comment:
I think you mean "write the test metadata"?

Reply:
Right, I was trying to copy the way we explained the constants, wanted to be consistent, forgot to change it. :) Thank you